polygeist/include/mlir/Conversion/PolygeistPasses.td

```diff
@@ -0,0 +1,53 @@
+#ifndef MLIR_CONVERSION_POLYGEISTPASSES
+#define MLIR_CONVERSION_POLYGEISTPASSES
+
+include "mlir/Pass/PassBase.td"
+
+def ConvertPolygeistToLLVM : Pass<"convert-polygeist-to-llvm", "mlir::ModuleOp"> {
+  let summary = "Convert scalar and vector operations from the Standard to the "
+                "LLVM dialect";
+  let description = [{
+    Convert standard operations into the LLVM IR dialect operations.
```

Comment:
ditto as above

Reply:
Deleted long description. Most dialects don't have a long description for such passes. We can explain we need bare-ptr conversion when we enforce it.